SPECS/mariner-rpm-macros/gen-ld-script.sh

```diff
@@ -9,8 +9,8 @@
 # /usr/lib/rpm/mariner/gen-ld-script.sh %{name} %{version}
 echo "gen-ld-script.sh name($1) version($2)"
 
-OS_ID=$(grep -oP '(?<=^ID=).+' /etc/os-release | tr -d '"')
-OS_VERSION=$(grep -oP '(?<=^VERSION_ID=).+' /etc/os-release | tr -d '"')
+OS_ID=$(sed -En '/^ID=/ s/.+="?([^"]+)"?/\1/ p' /etc/os-release)
+OS_VERSION=$(sed -En '/^VERSION_ID=/ s/.+="?([^"]+)"?/\1/ p' /etc/os-release)
```

Comment:
Thanks for explaining this change and for catching that. Looks good to me. Here's a suggestion how to simplify it further:

```suggestion
OS_ID=$(sed -En 's/^ID="?([^"]+)"?/\1/p' /etc/os-release)
OS_VERSION=$(sed -En 's/^VERSION_ID="?([^"]+)"?/\1/p' /etc/os-release)
```